examples/basic/variables.tf

```diff
@@ -20,6 +20,11 @@ variable "prefix" {
   default     = "basic"
 }
 
+variable "instance_name" {
```

Comment:
do not expose this in the example. Examples should have the bare minimum of inputs. Use the prefix variable in the main.tf as part of instance naming